src/status_im/config.cljs

```diff
@@ -133,3 +135,11 @@
 (def community-accounts-selection-enabled? true)
 (def fetch-messages-enabled? (enabled? (get-config :FETCH_MESSAGES_ENABLED "1")))
 (def test-networks-enabled? (enabled? (get-config :TEST_NETWORKS_ENABLED "0")))
+
+(def default-wallet-connect-metadata
+  {:name "Status"
+   :description
+   "Status is a secure messaging app, crypto wallet, and Web3 browser built with state of the art technology."
+   :url "https://status.app"
+   :icons
+   ["https://res.cloudinary.com/dhgck7ebz/image/upload/f_auto,c_limit,w_1080,q_auto/Brand/Logo%20Section/Mark/Mark_01"]})
```

Comment:
not sure if makes sense to keep it in the config. Doesn't look like something that will be used outside the initialisation from `status-im.contexts.wallet.common.utils.wallet-connect` and might be easier to just keep them together.